### PR TITLE
Some stuffs.

### PR DESCRIPTION
**Pull request by dnqbob**

### CODE REVIEW

File: mods/gen/weapons/missiles.yaml [hunk 1]
```diff
@@ -602,13 +602,29 @@ missile.missile_defender.garrison.air:
...
+	Projectile: LaserZap
+		Width: 30
+		ZOffset: 2047
+		Color: FF0000
+		Duration: 3
 
 missile.missile_defender.locked.ground:
 	Inherits: missile.missile_defender.ground
-	ValidTargets: Vehicle, Tank, Ship
+	ValidTargets: Vehicle, Tank, Ship, Structure
```
Comment (MustaphaTR) [inline mods/gen/weapons/missiles.yaml]: I checked the original game, doesn't seem like you can lock on buildings, just vehicles. Why add Structure here?

File: mods/gen/audio/voices.yaml [hunk 2]
```diff
@@ -844,3 +844,8 @@ StealthFighterVoice:
 		Select: vstesea, vsteseb, vstesec, vstesed, vstesee, vstesef
 		Move: vstemoa, vstemob, vstemoc, vstemod, vstemoe
 		Action: vsteata, vsteatb, vsteatc, vsteatd, vsteate
+
+ArtyBarrageSound:
+	DefaultVariant: .wav
+	Voices:
+		Action: sartweaa.wav
```
Comment (MustaphaTR) [inline mods/gen/audio/voices.yaml]: Definition should't have the extension here, it'll be looking for `sartweaa.wav.wav`.

File: mods/gen/weapons/missiles.yaml (hunk 1 above)
Comment (dnqbob) [reply mods/gen/weapons/missiles.yaml]: really? then I will remove this

File: mods/gen/audio/voices.yaml (hunk 2 above)
Comment (dnqbob) [reply mods/gen/audio/voices.yaml]: thanks, going to fix that later.

**Comment by dnqbob** [comment]
All fixed

**Pull request merged**